proto/api.proto

```diff
@@ -424,6 +424,8 @@ message Trade {
   double price = 3;
   string orderID = 4;
   bool isMaker = 5;
+  string address = 6;
```

Comment:
This name is a bit confusing. A trade has no address of it's own so who's address is this ?
Not a big deal though